packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -71,8 +69,7 @@ class _DriverBinding extends BindingBase with ServicesBinding, SchedulerBinding,
 /// with an `isError` boolean.
 void enableFlutterDriverExtension({ DataHandler handler, bool silenceErrors = false }) {
   assert(WidgetsBinding.instance == null);
-  _DriverBinding(handler, silenceErrors);
-  assert(WidgetsBinding.instance is _DriverBinding);
+  ExtensionBinding.registerExtensionRegistrar(_DriverExtensionRegistrar(handler, silenceErrors));
 }
 
 /// Signature for functions that handle a command and return a result.
```

Comment:
nit: please fix the formatting before committing. You're missing spaces after `for`, around operators, etc.

Reply:
Where is `for` in this case?